### ISSUE TITLE
Synology Package availible

### ISSUE THREAD

**Issue by gnadelwartz** [webmin/webmin#754]
to install webmin on Synology Disc Station I build a SPK to install on webmin on Synology Disc Station.

See: https://github.com/gnadelwartz/synomin

**Comment by gnadelwartz** [comment]
BTW: When will 1.872 be released? the webmin latest package is 1.870

**Comment by chris001** [comment]
Interesting!  Would be also nice to add a script to build a virtualmin SPK.

**Comment by gnadelwartz** [comment]
in theory this can be done, but lm not shure if virtualmin will run on an embedded system like Synology NAS.

**Comment by chris001** [comment]
Yes, virtualmin should run fine on Synology NAS, because it's simply perl scripts.

**Comment by jcameron** [comment]
There's a 1.872 devel version available at http://www.webmin.com/devel.html

**Comment by gnadelwartz** [comment]
@chris001 virtualmin runs only on the major distrubutions because it checks and install also other software. 

it may possible to install the perl scripts by hand on synology, but the real work is to check out the needed programs which are not installed and if availible they are nstalled in unusual locations. this was/is also the major work with webmin, figure out configuration and work around not existing software.

when webmin support for synology is mostly done lll have a look at virtualmin.

**Comment by gnadelwartz** [comment]
@swelljoe you have 1.872 of webmin as current  in http://software.virtualmin.com/gpl/wbm/ 

can i use your webmin-current.tar.gz as stable download link for my synology install script to have always the latest "stable" dev?

**Comment by qooob** [comment]
Kay, it's not always the case. Most of the time, webmin.com repo gets recent versions first.
